pkgs/tools/security/amber/default.nix

```diff
@@ -25,6 +25,7 @@ rustPlatform.buildRustPackage rec {
   meta = with lib; {
```

Comment:
passthru.updateScript, nix-update-script

Reply:
> passthru.updateScript, nix-update-script

Can we stop adding these everywhere? I'm pretty sure the bot can handle most common cases even without an explicitly set updateScript.